install/gcp/installer/forseti_server_installer.py

```diff
@@ -101,16 +106,22 @@ def deploy(self, deployment_tpl_path, conf_file_path, bucket_name):
             instance_name = 'forseti-{}-vm-{}'.format(
                 self.config.installation_type,
                 self.config.identifier)
-            self.wait_until_vm_initialized(instance_name)
 
             # Create firewall rules.
             self.create_firewall_rules()
+            self.wait_until_vm_initialized(instance_name)
+
+            # Delete firewall rules.
+            self.delete_firewall_rules()
 
         return success, deployment_name
 
     def create_firewall_rules(self):
         """Create firewall rules for Forseti server instance."""
-        # Rule to block out all the ingress traffic.
+        # Rule to block out all the ingress traffic on server VM.
+        # We need a service account deny for ingress that is more
```

Comment:
How does this rephrase look to you?
```
        # This rule overrides the implied deny for ingress
        # because we want a rule with a higher priority
        # that would be harder to be overridden.
```

Reply:
We need to mention the second reason as well, changed to below
# This rule overrides the implied deny for ingress
# because it has higher priority and is specific to service account.

Reply:
We need to mention the second reason as well, changed to below
   This rule overrides the implied deny for ingress
   because it has higher priority and is specific to service account.